professionalism/difficult-workplace-situations.md

```diff
@@ -8,7 +8,17 @@ About 1 hour
 
 ### Motivation
 
-You will definitely have difficult conversations in your career! Learn here some tips for successfully navigating through those tough times.
+You will definitely have difficult conversations in your career! Hence it is important to learn here some tips for successfully navigating through those tough times.
+
+**Examples of some difficult situation one has to deal with in the workplace:**
+- You’re Overwhelmed With Work
```

Comment:
the capitalization is a bit distracting. Either:
1. only capitalize the first letter or each line
or
2. provide the same links for each line that are provided in the article.